src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutAddressingContext.php

```diff
@@ -294,6 +294,14 @@ public function iSpecifyThePasswordAs($password)
         $this->addressPage->specifyPassword($password);
     }
 
+    /**
+     * @Then I should see :displayName in checkout header
```

Comment:
`displayName` -> `purchaserName`?

Reply:
thats the problem,  wasnt sure how to name it's email or full name :/